README.md

```diff
@@ -68,3 +68,17 @@ Commands:
 * Format: `black .`
 * Sort imports: `isort .`
 * Lint file: `pylint [path-to-file.py]`
+
+### Migrations
+
+This project is Using [Alembic](https://alembic.sqlalchemy.org/en/latest/) for migrations. To
+install alembic in your project just run `pip install alembic`
+
+Commands to use alembic:
+
+* Initialize: `alembic init alembic`
+* Migration Script: `alembic revision -m "migration scrip name"`
```

Comment:
What is this for?

Reply:
this command generate an empty version script just with version id, updgrade and downgrade method

Reply:
Could document it better? It is not clear to know that

Reply:
DONE,